src/Agent/NewRelic/Agent/Core/Metrics/MetricNameService.cs

```diff
@@ -24,8 +24,6 @@ public class MetricNameService : ConfigurationBasedService, IMetricNameService
         private static readonly TransactionMetricName NormalizedWebTransactionMetricName = new TransactionMetricName(MetricNames.WebTransactionPrefix, "Normalized/*");
```

Comment:
These 2 "Normalized..." vars are no longer needed either.

Reply:
Thanks for catching that.  Yay, even more cleanup!